Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/FRFSTicketService.hs

```diff
@@ -927,29 +1022,63 @@ getFrfsAutocomplete (_, mId) mbInput opCity origin vehicle = do
                         }
                   )
                   stops
-          stopsDistanceResp <-
-            Maps.getFrfsAutocompleteDistances mId merchantOpCity.id $
-              GetDistancesReq
-                { origins = NonEmpty.fromList transformedStops,
-                  destinations = NonEmpty.fromList [origin],
-                  distanceUnit = Meter,
-                  sourceDestinationMapping = Nothing,
-                  travelMode = Just Maps.CAR
-                }
-          return $
-            map
-              ( \stopResp ->
-                  FRFSStationAPI
-                    { name = stopResp.origin.name,
-                      code = stopResp.origin.code,
-                      lat = Just stopResp.origin.lat,
-                      lon = Just stopResp.origin.lon,
-                      distance = Just stopResp.distance,
-                      stationType = Nothing,
-                      sequenceNum = Nothing,
-                      address = Nothing,
-                      color = Nothing,
-                      towards = Nothing
-                    }
+          results <-
+            mapM
```

Comment:
@Radhika1708 Here we are calling OSRM for each stop, that is not the ideal approach as many stops will be there and if each OSRM network call takes 100 ms then for 5000 stops in worst case it would take 50 seconds for the response, which is not optimal. 
Instead, here we will call OSRM in single call and if it fails then ignore the distance for now. Meanwhile lets try to benchmark OSRM locally with how many max number of points can it take at once, then we will call in that many batches.